network/discovery/dv5_service.go

```diff
@@ -157,7 +180,7 @@ func (dvs *DiscV5Service) Bootstrap(logger *zap.Logger, handler HandleNewPeer) e
 			return
 		}
 		handler(e)
-	}, defaultDiscoveryInterval) // , dvs.forkVersionFilter) //, dvs.badNodeFilter)
+	}, defaultDiscoveryInterval, dvs.ssvNodeFilter(logger), dvs.sharedSubnetsFilter(1), dvs.badNodeFilter(logger), dvs.alreadyConnectedFilter(), dvs.recentlyTrimmedFilter())
```

Comment:
using `ssvNodeFilter` here results in `TestDiscV5Service_Bootstrap` failing (I guess because we don't appropriately mark `NewTestingNode` as "ssv"),

not sure why we need this kind of filtering (should I try to adjust this failing test, or get rid of it) @y0sher can you elaborate ?

Reply:
Well, the SSV filter isn't even necessary after alan because we added protocol ID. should we just remove it cc: @moshe-blox ?

Reply:
or we can just mark it as `ssv` in `NewTestingNode`

Reply:
@moshe-blox could you comment which one ^ we want to go with ?